### PR TITLE
Minor spelling fixes.

### PR DESCRIPTION
**Pull request by zoq**
Minor spelling fixes, my editor auto-formatted some of the lines, let me know if I should revert the 80 column character formation.

Signed-off-by: Marcus Edel <marcus.edel@collabora.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,16 @@
   <video src="https://user-images.githubusercontent.com/107984/157506173-5d2788e1-71ca-49ae-afc3-b7f7afbab798.mp4"></video>
 </div>
 
-*We asked an [AI](https://colab.research.google.com/github/zippy731/disco-diffusion-turbo/blob/main/Disco_Diffusion_v5_Turbo_%5Bw_3D_animation%5D.ipynb) what it thinks about our SocketCAN Kubernetes plugin, this was it's answer...  
+*We asked an [AI](https://colab.research.google.com/github/zippy731/disco-diffusion-turbo/blob/main/Disco_Diffusion_v5_Turbo_%5Bw_3D_animation%5D.ipynb) what it thinks about our SocketCAN Kubernetes plugin, this was it's answer...
```
Comment (jpc) [inline README.md]: The double-space removal at the end is unintended, right?

For myself I disabled the automatic space removal at the end of line and instead installed a plugin that just highlights these spaces so I can decide myself. I guess disabling it for Markdown files would work as well.

**Comment by jpc** [comment]
The 80-character wrapping (or the justification I once used) makes it nicer to read in text form but completely ruins the diffs. :/

**Comment by zoq** [comment]
> The 80-character wrapping (or the justification I once used) makes it nicer to read in text form but completely ruins the diffs. :/

I reverted the format changes and just committed the minor spelling fixes.

**Comment by jpc** [comment]
Thanks a lot :)

**Pull request merged**